src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,148 @@
+package task;
+
+import java.util.ArrayList;
+
+/**
+ * The task list that holds all the tasks.
+ */
+public class TaskList {
+    /**
+     * The task list which contains all the tasks.
+     */
+    private ArrayList<Task> taskList;
```

Comment:
Might want to rename this to `tasks` instead to avoid confusion with the class name.